connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -429,9 +431,13 @@ private void convertMessages(ConsumerRecords<byte[], byte[]> msgs) {
                     msg.offset(),
                     ConnectUtils.checkAndConvertTimestamp(msg.timestamp()),
                     msg.timestampType());
+            log.trace("{} applying transformations to message with key {}, value {}", this, msg.key(), msg.value());
             record = transformationChain.apply(record);
             if (record != null) {
                 messageBatch.add(record);
+            } else {
+                log.trace("{} dropping message after transformations returned null for message with key {}, value {}",
```

Comment:
nit (sorry): would be nice to be consistent with capitalization of the first letter.